new/modals.css

```diff
@@ -0,0 +1,115 @@
+.modal {
+  position: fixed;
+  top: 0;
+  display: none;
+  z-index: 100000;
+  width: 100%;
+  height: 100%;
+  background: #c1c7d0;
+  opacity: 0.5;
+  background-color: black;
+}
+
+.modal-bg {
+  max-height: calc(100vh - 10px) !important;
+  min-height: 80vh;
+  height: 90%;
+  display: none;
+  position: fixed;
+  top: 3%;
+  overflow-y: auto !important;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 85%;
+  margin-left: 6%;
+  background-color: #fff;
+  padding: 0 16px;
+  border-radius: 16px;
+  z-index: 100001;
+}
```

Comment:
- [ ] Your modal is working fine, but it does not look responsive on the desktop view. if the browser is not full-screen, the image inside the modal can be stretched, and a scrollbar appears on the right. This is not good for the user experience. Please try to make the modal responsive and not make the image stretch.
 - [ ] On the mobile view, the darker background is not applied to the entire screen, leaving a white line on the left. Kindly fix the background issue so it can look consistent.
 - [ ] Also on the mobile view, due to the size of the elements and huge description, it is impossible to see the links to the live version and GitHub of the app. The user may not know that he/she has to scroll down to see it and it is not a good user experience to have to scroll inside a modal. Please try to change the font sizes and maybe reduce the description so everything can be displayed on the screen.
 -Large desktop view:
<img width="1465" alt="image" src="https://user-images.githubusercontent.com/61919617/189162362-0effc5c2-c049-46f6-b582-5d6ada079fb6.png">
-Mobile Iphone SE view:
<img width="482" alt="image" src="https://user-images.githubusercontent.com/61919617/189163446-9ba6a693-dfe5-4f26-a641-35d08933343b.png">